app/serializers/users.js

```diff
@@ -0,0 +1,5 @@
+exports.userSerializer = user => ({
```

Comment:
The response should be in snake_case, check the node good practices document

Reply:
Done!